app/controllers/companies_controller.rb

```diff
@@ -23,17 +23,23 @@ def index
 
     @search_params = Company.ransack(action_params)
 
+    # put the companies in the desired order
+    @order_companies_by = company_order
+
     # only select companies that are 'complete'; see the Company.complete scope
 
     @all_companies = @search_params.result(distinct: true)
                          .includes(:business_categories)
                          .includes(addresses: [:region, :kommun])
                          .joins(addresses: [:region, :kommun])
+                         .order(@order_companies_by)
```

Comment:
Were you intending to use the `@order_companies_by` instance var in a view?  If not, it is not needed.

Reply:
@weedySeaDragon Did you see my comment re this unneeded(?) instance var?

Reply:
@patmbolger I did see your comment and took it out in my latest commit.  https://github.com/AgileVentures/shf-project/pull/763/commits/0ff2eb0cee8c216a1e44ad35d86cba66679a6ad0
 Maybe you need to pull that latest commit?